apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryCommand.Data.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Data;
+using System.Data.Common;
+
+namespace Google.Cloud.BigQuery.V2
+{
+    public sealed partial class BigQueryCommand : DbCommand
```

Comment:
there will be some overlap of members due to name conflicts where original members become overrides.
If you don't mind, I'll move those over as well when I'm going thru it.  someone looking at this after its done wouldn't care about the distinction between methods that existed previously.